spec/fixtures/declarative_config.yaml

```diff
@@ -0,0 +1,25 @@
+_format_version: "1.1"
```

Comment:
I expect many different declarative config tests to exist in the test suite (I know I'm adding a bunch in my branch!), so a single yaml file called "the" `spec/fixtures/declarative_config.yml` will be confusing (or will grow huge as people reusing adding more stuff to test). 

Since this is not a shared file anyway, could you embed this file in the test where it is used (e.g. as a long-string that's dumped to a temp file and then use it instead)? It would be easier to read the test as well, cross-checking the config against the result. Something like
```lua
local function make_yaml_file(content)
  local filename = os.tmpname()
  os.rename(filename, filename .. ".yml")
  filename = filename .. ".yml"
  local fd = assert(io.open(yaml_file, "w"))
  assert(fd:write(content))
  assert(fd:close())
  return filename
end

it("my test", function()
  local yaml_file = make_yaml_file [[
    _format_version: "1.1"
    ...
  ]]
  finally(function()
    os.remove(yaml_file)
  end)
```